internal/xds/translator/listener.go

```diff
@@ -226,6 +229,18 @@ func buildXdsTCPListener(
 	return listener, nil
 }
 
+func xdsListenerName(name, address string, port uint32, useAddressAsListenerName bool) string {
+	if useAddressAsListenerName {
+		return fmt.Sprintf("%s-%d", address, port)
```

Comment:
can you share what the listener metric label looks like ?
does envoy replace the `.` in `0.0.0.0` ?
should we instead just use protocol-port instead ?

Reply:
Envoy doesn't use the listener name in its metrics name. 

```
 Every listener has a statistics tree rooted at listener.<address>. (or listener.<stat_prefix>. if stat_prefix is non-empty) with the following statistics.
 ```
 https://www.envoyproxy.io/docs/envoy/latest/configuration/listeners/stats

```
 listener.0.0.0.0_10080.downstream_cx_total: 2
 listener.0.0.0.0_10080.downstream_cx_transport_socket_connect_timeout: 0
 listener.0.0.0.0_10080.downstream_global_cx_overflow: 0
 ```